examples/complete/main.tf

```
@@ -126,7 +126,7 @@ module "efs_disabled" {
 
 module "vpc" {
   source  = "terraform-aws-modules/vpc/aws"
-  version = "~> 3.0"
+  version = "~> 4.0"
```

```suggestion
  version = "~> 5.0"
```

Version 5 is available.